chapter01_crashcourse/chapter-one-problem-set.ipynb

```diff
@@ -0,0 +1,215 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Problem Set \n",
+    "\n",
+    "\"For the things we have to learn before we can do them, we learn by doing them.\" - Aristotle\n",
+    "\n",
+    "There's nothing quite like working with a new tool to really understand it, so we have put together some exercises through this book to give you a chance to put into practice what you learned in the previous lesson(s). "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Problems using ndarray [(Official Documentation)](https://mxnet.incubator.apache.org/api/python/ndarray.html) \n",
+    "\n",
+    "\n",
+    "Problem 1: Initialize an ndarray of dimension 1x256 on the GPU without overwriting its memory. Then, find the index of the maximum value in the array (argmax)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Problem 1 Work Area"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Problems from Linear Algebra\n",
+    "\n",
+    "Problem 2: Create a 4x4 matrix of random value. Then create an 4x4. identity matrix (an identity of size n is the n × n square matrix with ones on the main diagonal and zeros elsewhere). Multiple the two together and verify that you get the original matrix back."
```

Comment:
of random value*s*. -- I'd also be more specific, there are many ways to initialize random values. maybe uniformly random on the interval [0,1]?

Reply:
"multiple the two" -> multiply the two